### PR TITLE
Docker Container Dev Guide added to Readme

### PR DESCRIPTION
**Pull request by ez2torta**
Added Readme and example files for easy docker container dev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,55 @@ Contributions to this project are encouraged and welcomed! Your expertise in cod
 A successful build will display "Project built :)".
 
 
+## Docker Container Development
+
+This needs to have Wine32 Installed on your Linux machine prior to running this (at least in the current version of this README file)
```
Comment (lhsazevedo) [inline README.md]: Wine32 is not need when using the docker container :)

```suggestion
```

**Review by lhsazevedo — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ez2torta) [reply README.md]: the docker container already has wine32 there?

Comment (lhsazevedo) [reply README.md]: Yup, see https://github.com/lhsazevedo/tokyo-bus-guide-decomp/blob/main/docker/Dockerfile#L22C1-L25C17

Comment (ez2torta) [reply README.md]: done!

**Comment by lhsazevedo** [comment]
Thank you and welcome!

**Pull request merged**